packages/browser-utils/src/metrics/inp.ts

```diff
@@ -83,7 +84,9 @@ function _trackINP(): () => void {
     const activeSpan = getActiveSpan();
     const rootSpan = activeSpan ? getRootSpan(activeSpan) : undefined;
 
-    const routeName = rootSpan ? spanToJSON(rootSpan).description : undefined;
+    // If there is no active span, we fall back to look at the transactionName on the scope
```

Comment:
Hm, this may produce a lot of garbage data when the user does custom instrumentation. Normally I would say this is fine but "garbage data" in this case would be polluting the INP dashboard in Sentry massively with random crap and people have no way of figuring out why.

Could we just store the last active route on some global and read it here? That way we will never not receive route/url names - never ending up with crap in the INP dashboard.

Reply:
but this is exactly the semantics of the transactionName on the scope, isn't it? we use this literally to set this as "transaction" on the INP span :D makes sense to me to use the same thing we set as transaction on e.g. errors here?

Reply:
Ah I am sorry. That is correct. Mentally I was still in the old world where we had transactions on the scope 😄 All good then!